parent/pom.xml

```diff
@@ -25,25 +25,27 @@
 
     <properties>
         <astyanax.version>3.8.0-bv8</astyanax.version>
-        <dropwizard-metrics-datadog.version>1.1.1</dropwizard-metrics-datadog.version>
+        <!--TODO: We need 2.x for dependency compatibility. Upgrade once a non-RC version becomes available-->
+        <dropwizard-metrics-datadog.version>2.0.0-RC1</dropwizard-metrics-datadog.version>
         <cassandra.version>2.2.4</cassandra.version>
-        <curator.version>2.4.0</curator.version>
-        <curator-extensions.version>1.4.1</curator-extensions.version>
-        <dropwizard.version>0.7.1</dropwizard.version>
-        <guava.version>16.0.1</guava.version>
+        <curator.version>4.0.1</curator.version>
+        <curator-extensions.version>2.0.0</curator-extensions.version>
+        <zookeeper.version>3.5.4-beta</zookeeper.version>
```

Comment:
I cannot recall off the top of my head what libraries were the problem, but it was impossible to get the older ZK library because of other dependency conflicts.

Reply:
but in this case we have inconsistency between server and client versions

Reply:
AFAIK, zookeeper is natively version agnostic.

Reply:
have you tested with zokeeper version 3.4.x?

Reply:
Early in the dev cycle i did and there were issues. I am reattempting it now.

Reply:
It appears to be working now 😄 . I will downgrade to 3.4.13

Reply:
👍